x/feedist/init_test.go

```diff
@@ -0,0 +1,58 @@
+package feedist
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"testing"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/store"
+)
+
+func TestGenesisKey(t *testing.T) {
+	const genesis = `
+		{
+			"feedist": [
+				{
+					"admin": "E94323317C46BDA2268FA3698BAF4F95B893E8C7",
+					"recipients": [
+						{"weight": 2, "address": "E94323317C46BDA2268FA3698BAF4F95B893E8C7"}
```

Comment:
maybe with two recipients?

also one case with invalid data... but I guess that is tested elsewhere

Reply:
Done in https://github.com/iov-one/weave/pull/316/commits/ed74004a0cac4ac17391afe7ef79614900659cdd 

The whole loading and validation are done by the model. Genesis is only parsing the JSON, nothing besides that, so there is not much to test as well. This is mostly a double check test to be safe that genesis works at all :smile: